Integrations/GoogleChronicle/GoogleChronicle.py

```diff
@@ -0,0 +1,1326 @@
+                                      suspicious_category_list.split(',')]
+
+    malicious_severity_list = demisto.params().get('override_severity_malicious')
+    suspicious_severity_list = demisto.params().get('override_severity_suspicious')
+    override_malicious_severity = malicious_severity_list if malicious_severity_list is not None else ''
+    override_suspicious_severity = suspicious_severity_list if suspicious_severity_list is not None else ''
+
+    override_malicious_confidence_score = demisto.params().get('override_confidence_score_malicious_threshold')
+    override_suspicious_confidence_score = demisto.params().get('override_confidence_score_suspicious_threshold')
+
+    malicious_confidence_score_threshold_str = demisto.params().get('override_confidence_level_malicious')
+    suspicious_confidence_score_threshold_str = demisto.params().get(
+        'override_confidence_level_suspicious')
+
+    override_malicious_confidence_score_str = malicious_confidence_score_threshold_str \
+        if malicious_confidence_score_threshold_str is not None else ''
+    override_suspicious_confidence_score_str = suspicious_confidence_score_threshold_str \
+        if suspicious_confidence_score_threshold_str is not None else ''
+
+    return {
+        'malicious_categories': override_malicious_categories,
+        'suspicious_categories': override_suspicious_categories,
+        'override_severity_malicious': override_malicious_severity,
+        'override_severity_suspicious': override_suspicious_severity,
+        'override_confidence_score_malicious_threshold': override_malicious_confidence_score,
+        'override_confidence_score_suspicious_threshold': override_suspicious_confidence_score,
+        'override_confidence_level_malicious': override_malicious_confidence_score_str,
+        'override_confidence_level_suspicious': override_suspicious_confidence_score_str
+    }
+
+
+def validate_configuration_parameters(param: Dict[str, Any]):
+    """
+    Check whether entered configuration parameters are valid or not.
+
+    :type param: dict
+    :param param: Dictionary of demisto configuration parameter
+
+    :return: raise ValueError if any configuration parameter is not in valid format else returns None
+    :rtype: None
+    """
+    # get configuration parameters
+    service_account_json = param.get('service_account_credential', '')
+    fetch_days = param.get('first_fetch_time_interval_days', '3 days').lower()
+    page_size = param.get('fetch_limit', '10')
+
+    try:
+        # validate service_account_credential configuration parameter
+        json.loads(service_account_json, strict=False)
+
+        # validate fetch_limit configuration parameter
+        if not page_size.isdigit():
+            raise ValueError('Incidents fetch limit must be a number')
+
+        # validate first_fetch_time_interval_days parameter
+        range_split = fetch_days.split(' ')
```

Comment:
@Itay4 - We don't expect the interval in minutes, hence for extra validation, splitting the range.

Reply:
not sure i understood what you’re saying here
you are sending the value of that parameter to `parse_date_range()` which validates the input, why is the extra validation needed?

Reply:
@Itay4 in input configuration for First fetch time interval. The time range to consider for initial data fetch.(<number> <unit>, e.g., 1 day, 7 days, 3 months, 1 year).  we are not allowing minutes and hours so we added a separate validation for it.

Reply:
@Itay4 - We dont want to allow inputs like "10 minutes" to go through. We expect that user enters starting from the range of hours. Since parse_date_range() would allow "10 minutes" as good, the extra validation would take care of it.